src/commands/trace/__tests__/trace.test.ts

```diff
@@ -0,0 +1,40 @@
+// tslint:disable: no-string-literal no-null-keyword
+import {TraceCommand} from '../trace'
+
+describe('trace', () => {
+  describe('signalToNumber', () => {
+    test('should map undefined to undefined', () => {
+      const command = new TraceCommand()
+      expect(command['signalToNumber'](null)).toBeUndefined()
+    })
+    test('should map SIGKILL to 137', () => {
```

Comment:
maybe use the constant you've defined for this? `badCommandExitCode`. Maybe `should map SIGKILL to $badCommandExitCode`

Reply:
This is a different value (137), not badCommandExitCode (127). 137 is 128 + 9 (which is the SIGKILL number)

Reply:
ops my bad

Reply:
I changed the expected value to `128 + 9` to make it more explicit